org-roam-capture.el

```diff
@@ -244,7 +244,7 @@ If the search is via title, it is assumed that the file does not
 yet exist, and Org-roam will attempt to create new file.
 
 If the search is via daily notes, 'time will be passed via
-`org-roam-capture--info'. This is used to alter the default time
+`org-roam-capture--info'.  This is used to alter the default time
```

Comment:
Can we disable double spacing haha

Reply:
#blamecheckdoc

I'll revert it.

Reply:
Reverted in 24b9e2f5a4464cb1e97121e06f7c9a8340d24732.